go/vt/vtgate/engine/ordered_aggregate_test.go

```diff
@@ -232,32 +222,6 @@ func TestOrderedAggregateGetFields(t *testing.T) {
 	assert.Equal(got, input)
 }
 
-func TestOrderedAggregateGetFieldsTruncate(t *testing.T) {
```

Comment:
not sure what we are trying to test here, but we have plenty of truncation tests already. this test was just confusing to me

Reply:
It is testing truncation on the GetFields call.
We should still keep it.